rust/src/connection/network/transmitter/rpc.rs

```diff
@@ -151,7 +153,28 @@ impl RPCTransmitter {
             Request::Transaction(transaction_request) => {
```

Comment:
We used to ignore errors in the "initial operation handler"  for transaction opening because we didn't know how to handle this in sync/async environment. However, these is a perfect place for these purposes: just don't return the result from the Transmitter until we get both responses.